src/ansys/fluent/core/fluent_connection.py

```diff
@@ -244,15 +244,9 @@ def __init__(
         if channel is not None:
             self._channel = channel
         else:
-            if not ip:
-                ip = os.getenv("PYFLUENT_FLUENT_IP", "127.0.0.1")
-            if not port:
-                port = os.getenv("PYFLUENT_FLUENT_PORT")
+            assert ip
+            assert port
```

Comment:
@prmukherj cc: @mkundu1 

This change corrects the order of execution, but now we are just raising low-level `AssertionError`s here. Note that @hpohekar has changes in this module where he has _raised_ the levels of the exceptions.

Perhaps we should incorporate the improved logic here into @hpohekar 's branch, and remove this change from this branch.

See https://github.com/ansys/pyfluent/pull/2176/files#diff-00c898f4e09231f8a736f0d6a5d8903afe4b6e4ae8a8a12b30b9234f8d70bddb

Reply:
@seanpearsonuk Updated error message here https://github.com/ansys/pyfluent/pull/2176/files.

Reply:
@seanpearsonuk Implemented improved logic here https://github.com/ansys/pyfluent/pull/2176/files#diff-00c898f4e09231f8a736f0d6a5d8903afe4b6e4ae8a8a12b30b9234f8d70bddb.

Reply:
@prmukherj can remove this change now.

Reply:
@prmukherj @seanpearsonuk  API codegen check is failing for this change.

![image](https://github.com/ansys/pyfluent/assets/106588300/7c059f3c-9b20-4c6f-9d58-33ce55209594)

https://github.com/ansys/pyfluent/actions/runs/6847153022/job/18614938386?pr=2176